app/Commands/Start.php

```diff
@@ -52,6 +52,18 @@ public function handle()
     protected function dockerComposePs()
     {
         return $this->runTask('Checking Docker', function () {
+            if ($this->artisanCallNoOutput('check-docker-version')) {
```

Comment:
Then here we can leave the check-docker-version commands to have their output, calling them without  using `artisanCallNoOutput`